pkg/deviceutils/fake-device-utils.go

```diff
@@ -17,12 +17,16 @@ package deviceutils
 import "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/resizefs"
 
 type fakeDeviceUtils struct {
+	skipResize bool
 }
 
-var _ DeviceUtils = &fakeDeviceUtils{}
+var _ DeviceUtils = &fakeDeviceUtils{
```

Comment:
Unnecessary newline, later fixed in separate commit with update-gofmt.sh so this change should not be here in this commit.

Reply:
This will merge w/ squash so this will go away.